invenio_github/templates/semantic-ui/invenio_github/settings/index.html

```diff
@@ -1,162 +1,169 @@
+              README will appear next to your repository below.') }}
+            </div>
+          </div>
+          <div class="three column row">
+            <div class="column centered">
+              <br />
+              <div class="ui toggle checkbox">
+                <input type="checkbox" checked>
+                <label></label>
+              </div>
+            </div>
+            <div class="column"></div>
+            <div class="column centered">
+              <br>
+              <a class="ui label blue" href="https://doi.org/10.5281/zenodo.8475">
+                <img src="{{ url_for('invenio_formatter_badges.badge', title='doi', value='10.5281/zenodo.8475', ext='svg') }}" alt="10.5281/zenodo.8475">
+              </a>
+              <br>
+              <small>{{ _("(example)") }}</small>
+            </div>
+          </div>
+        </div>
+      </div>
       {{ helpers.panel_end(with_body=False) }}
-    {%- endif %}
-  {%- endfor %}
-{% endif %}
+    {%- endblock %}
 
+    {%- if repos %}
+      {%- block enabled_repositories %}
+        {%- for repo_id, repo in repos if repo.instance and repo.instance.hook %}
+          {%- if loop.first %}
+            {{ helpers.panel_start(_('Enabled Repositories') , with_body=False) }}
+            <div class="ui segment">
+          {%- endif %}
+          <div class="repositories-list">
+            {% include "invenio_github/settings/index_item.html" with context %}
+          </div>
+          {%- if loop.last %}
+            </div>
+            {{ helpers.panel_end(with_body=False) }}
+          {%- endif %}
+        {%- endfor %}
+      {%- endblock %}
+    {% endif %}
 
-{{ helpers.panel_start(_('Repositories'), with_body=False) }}
-<div class="panel-body">
-  <p>
-    <small class="text-muted">
-    If your organization's repositories do not show up in the list, please
-    ensure you have enabled <a href="https://help.github.com/articles/approving-third-party-applications-for-your-organization/">third-party
-    access</a> to the {{config.THEME_SITENAME | default('Invenio')}} application.
-    Private repositories are not supported.
-    </small>
-  </p>
-</div>
-{%- if not repos %}
-<div class="panel-body">
-  <p>
-    You have no repositories on GitHub.<br>
-    Go to <a href="https://github.com/new">GitHub</a> and create your first or
-    click Sync-button to synchronize latest changes from GitHub.
-  </p>
-</div>
-{%- else %}
-  {%- for repo_id, repo in repos if not repo.instance or not repo.instance.hook %}
-    {%- if loop.first %}
-    <hr />
-    {%- endif %}
-    {% include "invenio_github/settings/index_item.html" with context %}
-  {%- endfor %}
-{% endif %}
-{{ helpers.panel_end(with_body=False) }}
+    {%- block disabled_repositories %}
+      {{ helpers.panel_start(_('Repositories') , with_body=False) }}
+      <div class="ui segment">
+        <p>
+          <small>
+            {{ _('If your organization\'s repositories do not show up in the list, please
```

Comment:
maybe adding a block here would be easier for people to override the text only...